main.c

```diff
@@ -426,10 +452,8 @@ static void parse_args(int argc, char *const argv[], struct options *options)
         case 'm':
             errno = 0;
             options->mtu = strtol(optarg, &strtol_e, 10);
-            if (errno || *strtol_e != '\0' || options->mtu <= 0 ||
```

Comment:
please keep this

Reply:
yes, I am still working on getting the tests passing.  I'll ping you when I've finished cleaned it up